### PR TITLE
Update ComboBox Style for Selection by Category Nodes.

### PR DESCRIPTION
**Pull request by ZiyunShang**
### Purpose

Update ComboBox Style for Selection by Category Nodes.

#### Current Status
SelectionCombo

#### New Status
SelectionCombo3

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers

@ShengxiZhang @wangyangshi @QilongTang 

### FYIs

@saintentropy @Amoursol

### CODE REVIEW

File: src/Libraries/RevitNodesUI/Controls/ComboControl.xaml [hunk 1]
```diff
@@ -31,7 +31,9 @@
         </Button>
         <ComboBox Grid.Row="1"
                   ItemsSource="{Binding Model.DropDownNodeModel.Items}"
-                  SelectedIndex="{Binding Model.SelectedIndex, Mode=TwoWay}"/>
+                  SelectedIndex="{Binding Model.SelectedIndex, Mode=TwoWay}"
+                  Style="{StaticResource RefreshComboBox}"
+                  Margin="0,0,0,0"/>
```
Comment (QilongTang) [inline src/Libraries/RevitNodesUI/Controls/ComboControl.xaml:36]: for all 0s, do we need this line?

**Review by QilongTang — COMMENTED** [review]
Just one comment, thanks for finding it out!

File: src/Libraries/RevitNodesUI/Controls/ComboControl.xaml (hunk 1 above)
Comment (ZiyunShang) [reply src/Libraries/RevitNodesUI/Controls/ComboControl.xaml:36]: Yes, we need this to reset the Margin value which set by RefreshComboBox here https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml#L1356

Comment (QilongTang) [reply src/Libraries/RevitNodesUI/Controls/ComboControl.xaml:36]: I see, thanks for the explanation!

**Comment by QilongTang** [comment]
I have triggered a build to post latest public nugets. They should be up in your morning

**Comment by ZiyunShang** [comment]
> I have triggered a build to post latest public nugets. They should be up in your morning

Due to the issue mentioned in slack channel, the "select" button style in new libraries has been broken down, so I will merge this first to fix DropDown List in "SelectionCombo".

**Pull request merged**